dags/blocksec_plugin/contract_interaction_operator.py

```diff
@@ -2,20 +2,22 @@
 from airflow.utils.decorators import apply_defaults
 from blocksec_plugin.web3_hook import Web3Hook
 from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
-from web3.exceptions import InvalidAddress
+from blocksec_plugin.ethereum_transaction_confirmation_sensor import EthereumTransactionConfirmationSensor
+from web3.exceptions import InvalidAddress, TransactionNotFound
 from constants.constants import PriceConstants
 
 class ContractInteractionOperator(BaseOperator):
     """
     Executes a generalized contract interaction
     """
 
-    template_fields = ['contract_address', 'function', 'args', 'abi_json']
+    template_fields = ['contract_address', 'function', 'args', 'abi_json', 'transaction_hash']
 
     @apply_defaults
     def __init__(self,
                  contract_address,
                  abi_json,
+                 confirmations=3,
```

Comment:
I think you didn't fully understand what this is, you create self.confirmations, but can you see where this gets used?